extensions/applicationinsights-debugplugin-js/scripts/setActiveCdnVersion.ps1

```diff
@@ -0,0 +1,678 @@
+[CmdletBinding()]
```

Comment:
And again also mostly a copy from the AISKU -- changed to support only the ext/ folder and ai.dbg. prefix